admob/main.go

```diff
@@ -114,7 +114,7 @@ func Verify(cbUrl *url.URL) (err error) {
 		return err
 	}
 
-	rawQuery := cbUrl.RawQuery
+	rawQuery := strings.Replace(cbUrl.RawQuery, "%20", " ", -1)
```

Comment:
Please change this line with
```golang
rawQuery, err := url.QueryUnescape(cbUrl.RawQuery)
if err != nil {
    return err
}
```

Reply:
That certainly makes sense. Why didn't I think of that :-) Hereby done!